projects/kubernetes/autoscaler/1-23/helm/schema.json

```diff
@@ -0,0 +1,151 @@
+{
+  "affinity": {
+  },
+  "autoDiscovery": {
+    "clusterName": null,
+    "tags": [
+      "k8s.io/cluster-autoscaler/enabled",
+      "k8s.io/cluster-autoscaler/{{ .Values.autoDiscovery.clusterName }}"
+    ],
+    "roles": [
+      "worker"
+    ],
+    "labels": [
+
+    ]
+  },
+  "autoscalingGroups": [
+
+  ],
+  "autoscalingGroupsnamePrefix": [
+
+  ],
+  "clusterAPIMode": "incluster-incluster",
+  "clusterAPIKubeconfigSecret": "",
+  "clusterAPIWorkloadKubeconfigPath": "/etc/kubernetes/value",
+  "clusterAPICloudConfigPath": "/etc/kubernetes/mgmt-kubeconfig",
+  "clusterAPIConfigMapsNamespace": "",
+  "cloudConfigPath": "",
+  "cloudProvider": "clusterapi",
+  "containerSecurityContext": {
+  },
+  "deployment": {
+    "annotations": {
+    }
+  },
+  "dnsPolicy": "ClusterFirst",
+  "expanderPriorities": {
+  },
+  "priorityConfigMapAnnotations": {
+  },
+  "extraArgs": {
+    "logtostderr": true,
+    "stderrthreshold": "info",
+    "v": 4
+  },
+  "extraEnv": {
+  },
+  "extraEnvConfigMaps": {
+  },
+  "extraEnvSecrets": {
+  },
+  "envFromConfigMap": "",
+  "envFromSecret": "",
+  "extraVolumeSecrets": {
+  },
+  "extraVolumes": [
+
+  ],
+  "extraVolumeMounts": [
+
+  ],
+  "fullnameOverride": "",
+  "image": {
+    "sourceRegistry": "public.ecr.aws",
+    "repository": "kubernetes/autoscaler",
+    "digest": "sha256:372064f0f0ef50ee025109fff55683e50d488b8ae5fe1cb62a7bbc0fc16a64ee",
```

Comment:
I'm guessing I should pull out this specific digest, and that the packages controller will populate it with the most current version?

Reply:
We'll have a specific schema.json per chart, but yeh, we shouldn't be validating a particular digest in there here